packages/react-components/react-combobox/src/components/Listbox/useListboxStyles.styles.ts

```diff
@@ -22,14 +23,25 @@ const useStyles = makeStyles({
     ...shorthands.padding(tokens.spacingHorizontalXS),
     rowGap: tokens.spacingHorizontalXXS,
   },
+
+  standaloneRoot: {
+    [`&:not(:focus) [${ACTIVEDESCENDANT_FOCUSVISIBLE_ATTRIBUTE}]::after`]: {
```

Comment:
I'm not a fan of this selector and how it handles the focus outline
* `:not` is not supported in our browser support matrix
* it's dependent on how the focus outlines of its child are implemented
* The selector is most likely single use and creates a CSS rule that is not very reusable.

I would propose to create a feature in active descendant controller to toggle the attribute that is used for the focus outline

```tsx
activeDescendantController.disableFocusVisible()
// before
<span data-activedescendant-focusvisible />

// after
activeDescendantController.enableFocusVisible()
<span data-activedescendant-focusvisible-inactive />
```

You could enable/disable the correct attribute inside focus/blur event listeners